pkg/internal/apis/certmanager/validation/certificate_test.go

```diff
@@ -488,6 +488,29 @@ func TestValidateCertificate(t *testing.T) {
 				field.Invalid(fldPath.Child("emailAddresses").Index(0), "mailto:alice@example.com", "invalid email address: mail: expected comma"),
 			},
 		},
+		"valid certificate with revision history limit == 1": {
+			cfg: &internalcmapi.Certificate{
+				Spec: internalcmapi.CertificateSpec{
+					CommonName:           "abc",
+					SecretName:           "abc",
+					IssuerRef:            validIssuerRef,
+					RevisionHistoryLimit: int32Ptr(1),
+				},
+			},
+		},
+		"invalid certificate with revision history limit < 1": {
+			cfg: &internalcmapi.Certificate{
+				Spec: internalcmapi.CertificateSpec{
+					CommonName:           "abc",
+					SecretName:           "abc",
+					IssuerRef:            validIssuerRef,
+					RevisionHistoryLimit: int32Ptr(0),
+				},
+			},
+			errs: []*field.Error{
+				field.Invalid(fldPath.Child("revisionHistoryLimit"), int32Ptr(0), "must not be less than 1"),
```

Comment:
`				field.Invalid(fldPath.Child("revisionHistoryLimit"), 0, "must not be less than 1"),`

Reply:
Do you think that without dereferencing there could be some issue?

(I can do that, but the test would be `field.Invalid(fldPath.Child("revisionHistoryLimit"), *int32Ptr(0), "must not be less than 1"),` since otherwise the types(`int`, `int32`) won't match so it wouldn't make it nicer as such)

Reply:
Ah yes, I guess it should be `int32(0)` for the passed value.

Reply:
haha yes, that didn't cross my mind somehow :)